docs/deploying.md

```diff
@@ -12,145 +12,123 @@ weight=3
 
 # Deploying a registry server
 
-You obviously need to [install Docker](https://docs.docker.com/installation/) (remember you need **Docker version 1.6.0 or newer**).
+You obviously need to [install Docker version 1.6.0 or newer](https://docs.docker.com/installation/).
 
-## Getting started
+## Running on localhost
 
 Start your registry:
 
-     $ docker run -d -p 5000:5000 \
-     		--restart=always --name registry registry:2
+    docker run -d -p 5000:5000 --restart=always --name registry registry:2
 
-That's it.
+You can now use it with docker:
 
-You can now tag an image and push it:
+	# Get any image from the hub and tag it to point to your registry:
```

Comment:
Can we separate the instructions and code blocks? 

e.g.

Get any image from the hub and tag it to point to your registry
```
docker pull ubuntu && docker tag ubuntu localhost:5000/batman/ubuntu
```

instead of 

```
# Get any image from the hub and tag it to point to your registry
docker pull ubuntu && docker tag ubuntu localhost:5000/batman/ubuntu
```